linux/DtaDevLinuxNvme.h

```diff
@@ -18,7 +18,13 @@ along with sedutil.  If not, see <http://www.gnu.org/licenses/>.
 
  * C:E********************************************************************** */
 #pragma once
-#include "linux/nvme.h"
+#include <linux/version.h>
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 4, 0)
```

Comment:
I propose to just update the kernel to 4.12, perhaps we can then stop distributing this patch.

Would that work?